packages/frontend-analytics/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "@govuk-one-login/frontend-analytics",
+  "version": "3.0.1",
+  "description": "Reusable GA4 package for GDS One Login",
+  "main": "lib/analytics.js",
+  "type": "module",
+  "exports": {
+    ".": "./lib/analytics.js"
+  },
+  "scripts": {
+    "build": "npm run prettier && webpack --config webpack.config.js --mode production",
+    "prettier": "prettier --write \"**/*.{js,jsx,ts,tsx,md}\"",
+    "eslint": "eslint .",
+    "precommit": "npm run eslint && npm run prettier",
+    "pub": "cp PACKAGE-README.md dist/README.md && cp package.json dist && cd dist && npm publish --access public",
+    "test": "jest",
+    "test:coverage": "jest --coverage src"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/govuk-one-login/di-fec-ga4.git"
```

Comment:
Should be pointed at the monorepo

There instances in this file:
Lines: 21, 34, 36

Reply:
Thanks, updated now 👍